### ISSUE TITLE
pingwatch webui does not show any data

### ISSUE THREAD

**Issue by aslmx** [fazalmajid/pingwatch#1]
Hi,

thanks for this awesome tool. I haven't fully tried it, but it looks to be exactly covering what i was searching for!

I got it working after struggling with go.

It does write values to pingwatch.sqlite just fine, but when i open the WebUi, the graph remains empty.

When i checked firefox debugging console, i see a lot of these messages

```
16:54:28.033 Couldn't parse "2020-09-19 16:50:10 +0200 CEST" as a date dygraph.min.js:3:12450
    x http://192.168.2.30:8086/dygraph.min.js:3
    parseCSV_ http://192.168.2.30:8086/dygraph.min.js:4
    loadedEvent_ http://192.168.2.30:8086/dygraph.min.js:4
    onreadystatechange http://192.168.2.30:8086/dygraph.min.js:4
```

Did i miss somethingß? How could i solve this?

thanks!

**Comment by fazalmajid** [comment]
Hmmm. Thanks for the report. It may be because my server is set to UTC, not a time zone. Let me test and get back to you.

**Comment by fazalmajid** [comment]
> On Sep 19, 2020, at 15:58, aslmx <notifications@github.com> wrote:
> 
> thanks for this awesome tool. I haven't fully tried it, but it looks to be exactly covering what i was searching for!
> 
> I got it working after struggling with go.

What exactly went wrong? Can I make the documentation, such as it is, clearer?

> It does write values to pingwatch.sqlite just fine, but when i open the WebUi, the graph remains empty.
> 
> When i checked firefox debugging console, i see a lot of these messages
> 
> 16:54:28.033 Couldn't parse "2020-09-19 16:50:10 +0200 CEST" as a date dygraph.min.js:3:12450
>     x http://192.168.2.30:8086/dygraph.min.js:3
>     parseCSV_ http://192.168.2.30:8086/dygraph.min.js:4
>     loadedEvent_ http://192.168.2.30:8086/dygraph.min.js:4
>     onreadystatechange http://192.168.2.30:8086/dygraph.min.js:4
> 
> Did i miss somethingß? How could i solve this?

So I confirmed this was caused by a value of the TZ environment variable other than UTC, and fixed it.

You will need to “git pull” and rebuild.

Thanks again for letting me know about this bug.

**Comment by aslmx** [comment]
Yeah thanks! It seems fixed. I kept pingwatch running over night.
Now after `git pull` and `go build` as per your instructions, i see values, also from yesterday. 

Regarding my problems: It was rather related to me not really knowing how go works. So i had some error messages when i first installed go, git pulled pingwatch and then tried to run go build. It was related to the sqlite3 library IIRC. I found some random instructions. Apparently i have not made the effort to write it down somewhere. It works now.
I'm not sure whether it is in your responsibility to cover this in your documentation.

I'll post some more Feature Requests in separate tickets, I hope you don't mind.
